packages/mdc-slider/index.js

```diff
@@ -163,6 +163,20 @@ class MDCSlider extends MDCComponent {
           this.ripple_.deactivate();
         },
         isRTL: () => getComputedStyle(this.root_).direction === 'rtl',
+        updateTickMarkClasses: (currentTickMark) => {
```

Comment:
Generally we want the adapter methods to have as little logic as possible, so I think we should move the `if`s and`for`s to the foundation and have the adapter provide 4 methods:
```
tickMarkHasClass: (tickMark, className) => tickMark.classList.contains(className),
tickMarkAddClass: (tickMark, className) => tickMark.classList.add(className),
tickMarkRemoveClass: (tickMark, className) => tickMark.classList.remove(className),
getTickMarks: () => this.tickMarkSet_.children
```

Reply:
I agree with this, but can we name them `addTickMarkClass`, etc. instead to be consistent with what we've done for other adapters in cases like this?

Reply:
Sounds good! Updated